CMakeLists.txt

```diff
@@ -36,6 +36,8 @@ endif()
 add_subdirectory (protocol)
 add_subdirectory (cubic-server)
 add_subdirectory (math)
+add_subdirectory (yaml-cpp)
```

Comment:
Including a whole library directly into our project is really not the proper way to do it, either have CMake download it and set it up or have it be here as a git-submodule (A CMake dependency would be better).